command/server/server.go

```diff
@@ -197,6 +198,14 @@ func setFlags(cmd *cobra.Command) {
 		"write all logs to the file at specified location instead of writing them to console",
 	)
 
+	cmd.Flags().BoolVar(
+		&params.useBls,
+		blsFlag,
+		false,
+		// TODO
```

Comment:
Leftover TODO

Reply:
This is not leftover but we need to update an interface of contract and the its implementation.